src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-container {
+  margin-left: 50px;
+}
+
+.logo {
+  display: flex;
+  padding: 10px 0;
+}
+
+.nav {
+  font-family: Roboto, Avenir, sans-serif;
+  font-size: 12px;
+  margin-right: 50px;
+  flex-shrink: 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  flex-direction: column;
+  padding: 23px 0;
+  list-style: none;
+  text-decoration: none;
+  color: black;
+  text-transform: uppercase;
+}
+
+.is-active {
+  color: #00acdc;
+  padding-bottom: 19px;
```

The same here; use numbers that are divided by 4. For example: 4, 8 ,12 ,16, 20 etc.
```suggestion
  padding-bottom: 20px;
```

---

if i change padding and padding-bottom, header`s height will be 62 px(